helm/conjur-config-cluster-prep/templates/golden_configmap.yaml

```diff
@@ -22,9 +22,7 @@ data:
     authnK8sClusterRole: {{ .Values.authnK8s.clusterRole.name | default "conjur-clusterrole" }}
```

Comment:
I think we want to remove the conditional for `authnK8sClusterRole:` field as well.